viper.go

```diff
@@ -1971,12 +1971,6 @@ func (v *Viper) searchInPath(in string) (filename string) {
 		}
 	}
 
-	if v.configType != "" {
-		if b, _ := exists(v.fs, filepath.Join(in, v.configName)); b {
```

Comment:
When the program starts looking for the current directory, `v.configType` not `""`. The executable file is same as `v.configName`, `b` is `true`.